installation/installing-images/README.md

```diff
@@ -32,6 +32,8 @@ You will need to use an image writing tool to install the image you have downloa
 - Select the SD card you wish to write your image to.
 - Review your selections and click 'Flash!' to begin writing data to the SD card.
 
+**Note**: for Linux users, _zenity_ might need to be installed on your machine for _balenaEtcher_ to be able to write the image on your SD card.
```

Comment:
I suspect you wanted to use backticks here
```
`
```
instead of underscores?
```
_
```
i.e. `zenity` and not _zenity_ :slightly_smiling_face:

Reply:
The underscores were intended here. I wanted to style it the same way as _balenaEtcher_ was already highlighted in the document.
But if you guys prefer using backticks for this I can go with it no problem! :)

Reply:
ping @JamesH65

Reply:
Hmm, we usually use backticks for command line text or similar. Not sure about references to packages.  Will have a think.

Reply:
I've pushed to copy edit - they can decide!